### PR TITLE
add feed for [solidot.org]

### PR DESCRIPTION
**Pull request by sgqy**

### CODE REVIEW

File: routes/solidot/main.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            .replace(/时/g, ':')
+            .replace(/[日分]/g, '');
+        let date = new Date(date_str);
+        date.setHours(date.getHours() - 8);
+        date = new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate(), date.getHours(), date.getMinutes(), date.getSeconds()));
+        const item = {
+            title: title,
+            author: author,
+            pubDate: date.toUTCString(),
+            category: category, // not supported: 2018-10-29
```
Comment (sgqy) [inline routes/solidot/main.js]: issue: 希望生成结果里添加category标签

Comment (DIYgod) [reply routes/solidot/main.js]: pr welcome

**Comment by DIYgod** [comment]
Merged, thanks!

File: routes/solidot/main.js (hunk 1 above)
Comment (sgqy) [reply routes/solidot/main.js]: will try it later :)

**Pull request merged**